components/menu/index.css

```diff
@@ -431,7 +439,8 @@ governing permissions and limitations under the License.
   }
 }
 
-.spectrum-Menu-item:focus-visible {
+.spectrum-Menu-item:focus-visible,
+.spectrum-Menu-back:focus-visible {
```

Comment:
Do we need to include `.is-keyboardFocused` here or are we moving forward with only using `:focus-visible` since browser support is good for that now?

Reply:
AFAIK `is-keyboardFocused` should only be needed where `:focus-visible` does not get us the desired effect. Mostly it's just needed on text input fields that use the keyboard, because they show `:focus-visible` on click per the spec.